corehq/apps/domain/static/domain/js/billing_statements.js

```diff
@@ -0,0 +1,129 @@
+/* global Stripe */
+hqDefine("domain/js/billing_statements", function() {
+
+    var initialPageData = hqImport("hqwebapp/js/initial_page_data"),
+        paymentMethodHandlers = hqImport("accounting/js/payment_method_handler"),
+        CRUDPaginatedList = hqImport("hqwebapp/js/crud_paginated_list");
+
+    $(function () {
+        Stripe.setPublishableKey(initialPageData.get("stripe_options").stripe_public_key);
+        var WireInvoiceHandler = paymentMethodHandlers.WireInvoiceHandler;
+        var PaymentMethodHandler = paymentMethodHandlers.PaymentMethodHandler;
+        var Invoice = paymentMethodHandlers.Invoice;
+        var TotalCostItem = paymentMethodHandlers.TotalCostItem;
+        var pagination = initialPageData.get("pagination");
+        var paginatedListModel = new CRUDPaginatedList.CRUDPaginatedListModel(
+            pagination.total,
+            pagination.limit,
+            pagination.page,
+            {
+                statusCodeText: pagination.status_codes,
+                allowItemCreation: initialPageData.get('item_creation_allowed'),
+                createItemForm: pagination.create_item_form,
+            }
+        );
+
+        $(function () {
```

Comment:
This is already inside a document ready handler (same comment a couple further down in this file).